storage/container_storage_interface/osd-persistent-storage-aws-efs-csi.adoc

```diff
@@ -9,31 +9,31 @@ toc::[]
 
 [IMPORTANT]
 ====
-This procedure is specific to the Amazon Web Services Elastic File System (AWS EFS) CSI Driver Operator, which is only applicable for {product-title} 4.10 and later versions.
+This procedure is specific to the Amazon Web Services Elastic File System (Amazon EFS) CSI Driver Operator, which is only applicable for {product-title} 4.10 and later versions.
```

Comment:
I have updated on Line 12. Please let me know if this instance needs to be revoked. Thank you!

Reply:
Updated, "This procedure is specific to the link:https://github.com/openshift/aws-efs-csi-driver-operator[AWS EFS CSI Driver Operator] (a Red Hat operator),......."